common/data_refinery_common/utils.py

```diff
@@ -358,7 +402,7 @@ def load_blacklist(blacklist_csv: str = "config/RNASeqRunBlackList.csv"):
         for line in reader:
```

Comment:
Same comment as above about the next() question but I guess this answers it & this is a pattern we use in our code.

Reply:
yeah I had only planned on fixing the pep8 issues but we I can open a PR, honestly since its being repeated we can with a single function